website/versioned_docs/version-2.0/intents/overview.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_label: Intents Overview
+title: Intents Overview
+hide_title: true
+original_id: overview
+---
+
+# Intents Overview
+
+Extending APIs from one application to another is powerful. However, it requires bi-lateral agreements where implementors build to proprietary APIs. A standard language for interaction between applications allows us to create workflows on the fly, so that applications can discover and link to another without any prior knowledge.
+
+
+FDC3 Intents define a standard set of verbs that can be used to put together common cross-application workflows on the financial desktop.
+* Applications register the Intents & [Context Data](context/overview) combinations they support in the [App Directory](app-directory/overview)
```

Comment:
```suggestion
* Applications register the Intents & [Context Data](../context/spec) combinations they support in the [App Directory](../app-directory/overview)
```

The above links were broken, contributing to some of the build errors for this PR. 

I have tested this suggestion locally.

Reply:
fixed